lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -6,11 +6,13 @@
                 .replace(/>/g, '&gt;')
         }
     %>
-    <%= interpolate_text('You are taking "{exam_link}" as a {exam_type} exam. The timer on the right shows the time remaining in the exam.', {exam_link: "<a href='" + exam_url_path + "'>"+gtLtEscape(exam_display_name)+"</a>", exam_type: (!_.isUndefined(arguments[0].exam_type)) ? exam_type : gettext('timed')}) %>
-    <span id="turn_in_exam_id" class="pull-right">
-        <span id="turn_in_exam_id">
+    <div class='exam-text'>
+        <%= interpolate_text('You are taking "{exam_link}" as a {exam_type} exam. The timer on the right shows the time remaining in the exam.', {exam_link: "<a href='" + exam_url_path + "'>"+gtLtEscape(exam_display_name)+"</a>", exam_type: (!_.isUndefined(arguments[0].exam_type)) ? exam_type : gettext('timed')}) %>
+    </div>
+    <div id="turn_in_exam_id" class="pull-right">
```

Comment:
If you need to style this element differently that other pull-rights, add another class.